recipe/meta.yaml

```
@@ -20,33 +21,40 @@ requirements:
     - python
     - pip
     - hatchling
+    - hatch-vcs
   run:
     - python
-  run_constrained:
-    - colorama >=0.4.6
+    - importlib_metadata >=4.6.0
```

```suggestion
    - importlib_metadata >=4.6.0  # [py<310]
```
https://github.com/PyCQA/isort/blob/6.1.0/pyproject.toml#L47